src/style.css

```
@@ -0,0 +1,148 @@
+/* #region Normalize */
+
+body {
+  margin: 0;
+}
+
+/* #endregion */
+
+html {
+  font-family: "Roboto", serif, sans-serif;
+}
+
+/* #region Stars */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(\images\star.svg);
+  background-position: center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(n + 1) {
+  background-image: url(\images\star-active.svg);
+}
+
+/* #endregion */
+
+/* #region Card content */
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 198px;

```

```suggestion
  width: 200px;

```
use box-sizing: border-box;

and checklist 7 https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md